app/assets/javascripts/application.js

```diff
@@ -33,6 +33,6 @@
 //= require configure_quill_editor
 //= require plugins_toggle
 
-//= require 'page'
+//= require('page');
```

Comment:
Ha, I figured you meant that's how it should be included. The error disappeared with it included like that.